drupal/recipes/wunder_pages/config/field.field.paragraph.accordion_item.field_content_elements.yml

```diff
@@ -13,7 +13,7 @@ dependencies:
   module:
     - entity_reference_revisions
 _core:
-  default_config_hash: gI9oa4eX3GU6bmtnilvYYWzqTIBnFP4QmP2DrvuBGlA
+  default_config_hash: 9SX29NIrIS2WBAgKvkOv-XjEtpNbr0BtMk5u4ZHc6fE
```

Comment:
These changes are irrelevant, they are the product of me overriding these config files in the recipe... we could also revert this change to make this clearer... or we just ignore this :)